app/models/container_image.rb

```diff
@@ -1,10 +1,19 @@
+require 'MiqContainer/MiqContainer'
+
 class ContainerImage < ActiveRecord::Base
```

Comment:
These changes sound entirely Not Okay; `ContainerImage` is provider independent, and shouldn't contain the word "docker"... and it certainly shouldn't contain the K word.